salt/migrations/admin/post_migration/init.sls

```diff
@@ -0,0 +1,31 @@
+# This state only removes the grains and reenables the transaction-update.
+
+# It is split into another file, because the admin node also hosts the
+# salt-master - so it can not be rebooted and waited on as the salt-master will
+# also be rebooted.
+
+reenable-transactional-update-timer:
+  service.running:
+    - name: transactional-update.timer
+
+{% for grain in 'tx_update_migration_available', 'tx_update_migration_notes', 'tx_update_migration_mirror_synced', 'migration_in_progress' %}
```

Comment:
we should also not delete the `tx_update_migration_mirror_synced` grain here. it needs to be present always

Reply:
i also added the additional grain in velum, so we ned it here too `tx_update_migration_mirror_newversion`